dashboard/app/views/layouts/application.html.haml

```diff
@@ -39,7 +39,7 @@
         = render partial: 'layouts/header', locals: {script: @script, script_level: @script_level, level: @level, full_width: view_options[:full_width]}
       - page = yield
       - if view_options[:full_width]
-        .full_container{style: ('padding: 0px;' if view_options[:no_padding])}
+        .full_container
```

Comment:
We do still set padding: 0 in project.js. Not sure if there are other places that will be missed now.